ConservationInternational/resilienceatlas
# Fix/RA2-247-layer-ordering

Fix layer ordering in the legend of the map and embed
Fix reordering on drag and drop

## Testing instructions

Go to the map: You should see any layers on the same order on the map than on the legend. Also you should be able to reorder them.

The same for the journeys embed.

https://localhost:3000/journeys/7/step/6

https://localhost:3000/map?layers=%5B%7B%22id%22%3A96%2C%22opacity%22%3A1%2C%22order%22%3A2%7D%2C%7B%22id%22%3A212%2C%22opacity%22%3A1%2C%22order%22%3A1%7D%5D&zoom=4&center=%7B%22lat%22%3A0.4833927027896987%2C%22lng%22%3A6.943359374999999%7D&tab=layers

## Tracking

https://vizzuality.atlassian.net/browse/RA2-247?atlOrigin=eyJpIjoiZTMxMDI1YjEwNzRlNGQyYzlmODQzMGQxODJmYzBjOWYiLCJwIjoiaiJ9
